<!DOCTYPE html>
<html lang="en-us">
<head prefix="og: http://ogp.me/ns#">
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1" />
  <meta property="og:title" content=" Christelle Aluome &middot;  Statistical and Theoretical Ecology Group" />
  
  <meta property="og:site_name" content="Statistical and Theoretical Ecology Group" />
  <meta property="og:url" content="https://statntheorecol.github.io/members/christelle-aluome/" />
  
  
    <meta property="og:type" content="article" />
    
    <meta property="og:article:published_time" content="2019-02-15T10:04:13&#43;01:00" />
    
  

  <title>
     Christelle Aluome &middot;  Statistical and Theoretical Ecology Group
  </title>

  <link rel="alternative stylesheet" href="https://statntheorecol.github.io/css/bootstrap.min.css" />
  <link rel="stylesheet" href="https://statntheorecol.github.io/css/font-awesome.min.css" />
  <link rel="stylesheet" href="https://statntheorecol.github.io/css/main.css" />
  <link rel="stylesheet" href="https://statntheorecol.github.io/css/github.css" />
  <link rel="stylesheet" href="https://statntheorecol.github.io/css/color-theme.css" />
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Source+Sans+Pro:200,300,400" type="text/css">
  <link rel="shortcut icon" href="https://statntheorecol.github.io/images/wordcloud.png" />
  <link rel="apple-touch-icon" href="https://statntheorecol.github.io/images/wordcloud.png" />

  <link rel="preload" href="https://statntheorecol.github.io/images/banner.png" as style /> 


  

  

  
</head>
<body>
  
  <header class="global-header"  style="background-image:url(https://statntheorecol.github.io/images/banner.jpg)">
  
    <section class="header-text">
      <h1><a href="https://statntheorecol.github.io/">Statistical and Theoretical Ecology Group</a></h1>
      
      
        <a class="btn btn-default btn-home" href="https://statntheorecol.github.io/">
          <i class="fa fa-angle-left" aria-hidden="true"></i>
          &nbsp;Home
        </a>
      
      
      <div class="navbar-container">
        
          <a class="btn btn-default navbar-item" href="https://statntheorecol.github.io/posts">
            Blog
          </a>
        
          <a class="btn btn-default navbar-item" href="https://statntheorecol.github.io/publications">
            Publications
          </a>
        
          <a class="btn btn-default navbar-item" href="https://statntheorecol.github.io/research">
            Research themes
          </a>
        
          <a class="btn btn-default navbar-item" href="https://statntheorecol.github.io/members">
            Members
          </a>
        
          <a class="btn btn-default navbar-item" href="https://statntheorecol.github.io/">
            Home
          </a>
        
      </div>
      
      
      
      
      
      
      
      
      
      
      
    </section>
  </header>
  <main class="container">



<article>
  <header class="article-title">
    <h1 class="text-primary">Christelle Aluome</h1>
  </header>
  <div class="delimiter"></div>
  <section>
    <p><img align="left" src="../../images/caluome2.JPG" width="250"  style="margin: 10px 20px"/></p>

<p>I am a bio-informatics engineer with a particular expertise in data mining and database development.</p>

<p>Generally speaking, I am interested in everything linked with data mining: management, exploitation and analysis (especially for big volumes of data).
I used to work in vegetable genomics, in diverse projects related to data mining, which includes database design (associated with the development of a website), analysis of sequences from NGS techniques (<em>Populus nigra</em>, <em>Pisum sativum</em>), the development of procedure for genome reconstruction (<em>de novo</em> assembly) in the case of small computing resources, and the development of protocols for the selection of SNP which can be applied to genotyping chip.</p>

<p>For the Integrative and Theoretical Ecology group, I have set up a web interface of spatialized metadata to favor interdisciplinary search (<a href="https://data.cote.labex.u-bordeaux.fr/geonetwork/srv/fre/catalog.search#/home">Geocatalog</a>, which uses the European standard INSPIRE), and I now work mainly on two themes :</p>

<ul>
<li>The first topic concerns the exploratory analysis of long time series of birds in the Teich bird sanctuary.</li>
<li>I also work also wetlands within the framework of the <a href="http://itecologyresearch.com/epizhone/">EPIZHONE project</a>. I have worked on a synthesis document which lists the information available on existing data (human uses, ecology, hydrology, cartography,…) on the wetland zones in the Gironde Estuary and the Arcachon Bay.</li>
</ul>

<p>I also take part in the creation and exploitation of GIS cartographic resources.</p>

<p>Since May 2017, I have been an information system administrator for the research unit <a href="https://www6.bordeaux-aquitaine.inra.fr/ispa_eng/">ISPA</a> (for Atmosphere Plant Soil Interactions), which is part of INRA. I have joined the <a href="https://www.icos-ri.eu/">ICOS</a> project (Integrated Carbon Observation System), a European network that aims at monitoring the greenhouse gas balance in different ecosystems in and around Europe for 20 years. Within this research infrastructure, I coordinate leaf analyses and monitor the provisioning of resulting data to all partners. These analyses concern forests (pines, oak, beech trees&hellip;), meadows and cultivated lands (maize&hellip;) to determine their contents in elements such as phosphorus, nitrogen or carbon, and to estimate their change with time. I am also the website manager for <a href="https://icos-eco.fr/">ICOS-France</a>.</p>

<p><strong>Selected publications:</strong></p>

<p>Al-Yaari A., Dayau S., Chipeaux C., Aluome C., Kruszewski A., Loustau D. &amp; Wigneron J.-P. <a href="https://www.mdpi.com/2072-4292/10/11/1839/htm">The AQUI Soil Moisture Network for Satellite Microwave Remote Sensing Validation in South-Western France.</a> <em>Remote Sensing.</em> 2018, 10(11), 1839.</p>

<p>Aluome C, Aubert G, Alves Carvalho S, Le Paslier MC, Burstin J, Brunel D. <a href="https://dx.doi.org/10.1186/s13104-016-1903-z">De novo construction of a &ldquo;Gene-space&rdquo; for diploid plant genome rich in repetitive sequences by an iterative Process of Extraction and Assembly of NGS reads (iPEA protocol) with limited computing resources</a>. <em>BMC Research Notes</em>. 2016, 9, 81.</p>

<p>Faivre-Rampant P, Zaina G, Jorge V, Giacomello S, Segura V, Scalabrin S, Guérin V, De Paoli E, Aluome C, Viger M, Cattonaro F, Payne A, PaulStephenRaj P, Le Paslier MC, Berard A, Allwright MR, Villar M, Taylor G, Bastien C, Morgante M. <a href="https://dx.doi.org/10.1111/1755-0998.12513">New resources for genetic studies in Populus nigra: genome-wide SNP discovery and development of a 12k Infinium array.</a> <em>Molecular Ecology Resources</em>. 2016, 16(4), 1023-1036.</p>

<p>Tayeh N, Aluome C, Falque M, Jacquin F, Klein A, Chauveau A, Bérard A, Houtin H, Rond C, Kreplak J, Boucherot K, Martin C, Baranger A, Pilet-Nayel ML, Warkentin TD, Brunel D, Marget P, Le Paslier MC, Aubert G, Burstin J. <a href="https://dx.doi.org/10.1111/tpj.13070">Development of two major resources for pea genomics: the GenoPea 13.2 K SNP Array and a high‐density, high‐resolution consensus genetic map</a>. <em>The Plant Journal</em>. 2015, 84, 1257-1273.</p>

<p>Rouard M, Guignon V, Aluome C, Laporte MA, Droc G, Walde C, Zmasek CM, Périn C, Conte MG. <a href="https://dx.doi.org/10.1093/nar/gkq811">GreenPhylDB v2. 0: comparative and functional genomics in plants.</a> <em>Nucleic acids Research</em>. 2011, 39, 1095-1102.</p>

  </section>
  <div class="clearfix">
    
    
    <div class="pull-right">
      
    </div>
  </div>

</article>
<div class="delimiter"></div>

  </main>
  <footer class="container global-footer">
    <div class="copyright-note pull-left">
      
    </div>
    <div class="sns-links hidden-print">
  
  
  
  
  
  
  
  
  
  
  
  
</div>

  </footer>

  
  
  
  
<script data-no-instant>document.write('<script src="/livereload.js?port=1313&mindelay=10"></' + 'script>')</script></body>
</html>

